pkg/kubectl/cmd/testdata/edit/testcase-apply-edit-last-applied/test.yaml

```diff
@@ -6,7 +6,7 @@ args:
 outputFormat: yaml
 namespace: myproject
 expectedStdout:
-- service/svc1 edited
```

Comment:
was this bugged before?  Changing tests always looks weird.

Reply:
This is the expected output here(print the modified case with yaml output format) matches the print way we fixed.